README.md

```diff
@@ -0,0 +1,52 @@
+# jupiterone-aws-integration
+
+Contains scripts and instructions to configure your [JupiterOne](https://jupiterone.io/)
+AWS integration.
+
+## Setup
+
+You can set up the necessary AWS IAM role for JupiterOne using one of the
+following methods:
+
+## Launch a stack now!
+
+Just click this button to launch a CloudFormation stack for provisioning your
+JupiterOne AWS integration:
+
+[![Launch JupiterOne CloudFormation Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=jupiterone-integration&templateURL=https%3A%2F%2Fs3.amazonaws.com%2Flifeomic-prod-us-jupiter-aws-integration%2Fjupiterone-cloudformation.json)
+
+## Using AWS CLI
+
+```bash
+aws cloudformation create-stack --stack-name JupiterOneIntegration --template-url https://s3.amazonaws.com/lifeomic-prod-us-jupiter-aws-integration/jupiterone-cloudformation.json
```

Comment:
Once we have an API for fetching an external id for an account, we should create a script that fetches it and then passes it in as a parameter to this command.